sdk/core/core/src/az_log.c

```diff
@@ -27,38 +27,61 @@ void az_log_set_callback(az_log_message_fn az_log_message_callback)
   _az_log_message_callback = az_log_message_callback;
 }
 
-void _az_log_write(az_log_classification classification, az_span message)
+// _az_log_write_engine is a function private to this .c file; it contains the code to handle
+// _az_log_should_write & _az_log_write.
+//
+// If log_it is false, then the function returns true or false indicating whether the message
+// should be logged (without actually logging it). if log_it is false, then the function returns
+// true or false indicating whether the message should be logged (without actually logging it).
```

Comment:
```suggestion
// If log_it is false, then the function returns true or false indicating whether the message
// should be logged (without actually logging it).
```

Reply:
Thank you Victor, good catch, I'll check it in once CI finishes running the unit tests (want to know sooner, for myself)